### DISCUSSION TITLE
Results server url for uploading test results

### DISCUSSION THREAD

**Discussion by MohanPrasad-J** [linux-netdev/nipa#33] [Q&A]
Hi, I am trying to run and upload the hardware selftest result for my network device.
Currently, I have run the poller script and have my result folder with the result of my hardware selftest(desc, retcode & summary files).
I am trying to upload this hardware selftest result to this device results link: https://netdev.bots.linux.dev/devices.html.
I understand that pw_upload script is used for uploading the test results to the results server, however I don't see any documentations mentioning the URL for the results server. Without this URL I am not able to upload results/proceed further. Could anyone enlighten about where to push in order to be visible in the device result page.

**Comment by kuba-moo** [comment]
I updated the flow diagram a bit here: https://github.com/linux-netdev/nipa/wiki/Netdev-CI-system hopefully to make things clearer.
We will configure NIPA to fetch results from you on our end. But you need to give us a URL to where results are stored (specifically URL to the results.json-like manifest file).
